README.md

```diff
@@ -1,28 +1,182 @@
 # Velveteen [![Build Status](https://travis-ci.com/thoughtbot/velveteen.svg?branch=master)](https://travis-ci.com/thoughtbot/velveteen)
 
-🚧 A work in-progress 🚧
-
 Transform your background jobs into a real data pipeline with Velveteen.
 
-## Install
+## We've got a pipeline problem
+
+Background jobs are an inevitable addition for many Rails applications. They
+typically start with a need to send email, then quickly grow into a collection
+of interdependent data processing jobs that are yearning to be a pipeline.
+Velveteen aims to insert itself near the beginning, providing support to build
+out the pipeline in a maintainable way.
+
+## 🚧 A work in-progress 🚧
+
+Prior to version 1.0, minor version updates are susceptible to breaking changes.
+
+## Getting started
+
+Velveteen requires Ruby `>= 2.5` and access to RabbitMQ.
 
 Add the following line to Gemfile:
 
 ```ruby
 gem "velveteen"
 ```
 
-and run `bundle install` from your shell.
+Run the bundle command to install it.
+
+Define a worker:
+
+```ruby
+# do_something.rb
+require "velveteen"
+
+Velveteen::Config.exchange_name = "velveteen_development"
```

Comment:
Should we explain the relationship between exchanges and routing keys somewhere (or link to something that does)?

Reply:
Added a generic link to the tutorial in 90d31a80cb3322d360d22e66533e55a7a0076ed2.

I can see a future improvement that adds details about how Velveteen maps to RabbitMQ, but that's a bit more than I want to take on at the moment.